lib/index.ts

```diff
@@ -266,6 +267,7 @@ export async function performBuilds(
 	} = await initializeBuildMetadata(tasks, docker, tmpDir);
 
 	const images = await Bluebird.map(tasks, (task: BuildTask) => {
+		task.dockerOpts = { platform: task.dockerPlatform, ...task.dockerOpts };
```

Comment:
`task.dockerPlatform` could be `undefined`, so we could end up with:
```
    task.dockerOpts = { platform: undefined, ...task.dockerOpts };
```
So probably better code it as:
```
if (task.dockerPlatform) {
    task.dockerOpts = { platform: task.dockerPlatform, ...task.dockerOpts };
}
```